sound/soc/sof/topology.c

```diff
@@ -3355,7 +3355,6 @@ static int sof_route_load(struct snd_soc_component *scomp, int index,
 		sroute->private = connect;
 		sroute->src_widget = source_swidget;
 		sroute->sink_widget = sink_swidget;
-		sroute->setup = true;
```

Comment:
This was actually found and proposed change in my previous PR #2850, see it here:
https://github.com/thesofproject/linux/pull/2850/files#r619475449